docs/UserGuide.md

```diff
@@ -211,42 +206,54 @@ more companies on your own!
+
 
 **Step 2:** You should see the details of Twitter updated.
 
-Now that you have learnt how to edit the details of a company, you can go ahead and try editing the details of a few
-more companies on your own! Having learnt how to add, view and edit a company in LinkMeIn, you are left
-with one last step to master LinkMeIn.
+Go ahead and try editing the details of a few more companies on your own! 
 
 ## 4. Deleting a company
 
-After adding a few companies, you were unfortunately rejected from some of the companies and you no longer need to track
-them in LinkMeIn. You can delete the companies that you no longer need to track in LinkMeIn.
+After adding a few companies, you can delete the companies that you no longer need to track in LinkMeIn.
 
-**Step 1:** To delete Twitter, type `delete 6` in the command box and press Enter.
+**Step 1:** To delete DSTA, type `delete 3` in the command box and press Enter.
 
-**Step 2:** You should see the details of Twitter removed from LinkMeIn.
+**Step 2:** You should see the details of DSTA removed from LinkMeIn.
 
 Go ahead and try deleting the other companies too. If you wish to read up more on the delete command, you can do so
 [here](#deleting-a-company--delete).
 
-_Congratulations! You have completed the quick tour of LinkMeIn. You are now ready to use LinkMeIn to track your
-internship applications. LinkMeIn offers more than just the features that you have learnt in this tour.
+
+**Congratulations!** You have completed the quick tour of LinkMeIn. You are now ready to use LinkMeIn to track your
+internship applications. 
+- After completing the tour, you're all set to start adding your own entries! Feel free to clear the sample entries by
+simply typing 'clear' in the command box. If you'd like to learn more about the 'clear' command, check it out [here](#clear-all-entries-clear).
+
+LinkMeIn offers more than just the features that you have learnt in this tour.
 If you wish to learn more about the other features of LinkMeIn, you can do so
 [here](#current-features). You can also visit the [FAQ](#faq) to view the frequently asked questions if you need
-more help!_
+more help!
 
 [&uarr; Back to Table of Contents](#table-of-contents)
 
+---
+
 # Current Features
 
+LinkMeIn's feature can be categorised into 3 broad categories:
+
+-  [General Features](#general-features): You will find essential tools for navigating LinkMeIn. Allowing you to get help,
+clear all entries, and exit LinkMeIn.
+- [Company Management Features](#company-management-features): You will find a set of commands that enable you to **interact with a company entry**.
+Allowing you to add, view, edit, delete, and add remarks to companies.
+- [Company List Features](#company-list-features): You will find a set of commands that enable you to **interact with the list of companies**.
+Allowing you to find, list, filter, and sort companies.
```

Comment:
Suggestion: 

[General Features](https://chat.openai.com/#general-features): This section equips you with essential tools to navigate LinkMeIn effectively. Here, you can access help, clear all entries, and gracefully exit LinkMeIn.

[Company Management Features](https://chat.openai.com/#company-management-features): Dive into this section to **engage with individual company entries**. You have the power to add, view, edit, delete companies, and even add your own remarks to companies.

[Company List Features](https://chat.openai.com/#company-list-features): Unlock the potential of this section to **interact with the list of companies**. Explore options to find, list, filter, and sort companies, managing all companies with ease.